gpflow/kernels/mo_kernels.py

```diff
@@ -137,9 +154,13 @@ class LinearCoregionalization(IndependentLatent, Combination):
     Linear mixing of the latent GPs to form the output.
     """
     def __init__(self, kernels, W, name=None):
```

Comment:
What is this kernel's `output_dim`?...

Reply:
we care about num latents not output dims

Reply:
this PR commit will reflect this